executor/linux/secret.go

```diff
@@ -336,7 +341,8 @@ func (s *secretSvc) stream(ctx context.Context, ctn *pipeline.Container) error {
 	r := s.client.repo
 
 	// stream all the logs to the init step
-	l, err := s.client.loadStepLogs(s.client.init.ID)
+	l, err := step.LoadLogs(s.client.init, &s.client.stepLogs)
+	// l, err := s.client.loadStepLogs(s.client.init.ID)
```

Comment:
Can this be deleted?

Reply:
:doh: Yes you are spot on! 👍 

Sorry about that

Reply:
Fixed 👍